ci/test-core.json

```diff
@@ -26,11 +26,13 @@
     "client/stats/...",
     "client/structs/...",
     "client/taskenv/...",
+    "client/vaultclient/...",
     "command/agent/...",
     "command/ui/...",
     "helper/...",
     "internal/...",
     "jobspec/...",
+    "jobspec2/...",
```

Comment:
Looks like there are some test failures associated with this package in this branch.

Reply:
Ah yes, I was waiting for CI to finish and forgot to mark this as draft. I will get the tests fixed today.

Reply:
Tests fixed in https://github.com/hashicorp/nomad/pull/19959/commits/8c6396994d365050ebfb347340f1c04c2f1688d2 by backporting the relevant parts of https://github.com/hashicorp/nomad/pull/19024.